query/encode.go

```diff
@@ -5,14 +5,23 @@ import (
 	"net/http"
 
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/csv"
 )
 
-const DialectType = "no-content"
+const (
+	NoContentDialectType     = "no-content"
+	NoContentWErrDialectType = "no-content-with-error"
+)
 
-// AddDialectMappings adds the no-content dialect mapping.
+// AddDialectMappings adds the no-content dialects mappings.
```

Comment:
```suggestion
// AddDialectMappings adds the mappings for the no-content dialects.
```

Reply:
I'll do this in next PR, to avoid  CI steps 👍